### ISSUE TITLE
[ic-button]: Using `aria-controls` or `aria-owns` on `ic-button`

### ISSUE THREAD

**Issue by jd239** [mi6/ic-ui-kit#1012]
## Summary of the bug
At the moment, the make up of the `ic-button` involves the `button` element itself being within shadow DOM. Ideally, the `aria-controls` and `aria-owns` attributes would be placed on the button and the IDs will link to the elements the button controls. However, there will be a a11y violation as the shadow DOM will prevent the link between the two elements. This ticket is to find a way around that. 

## Additional info
- Is it possible to have `aria-owns` and `aria-controls` on the host tag instead?

**Comment by jd239** [comment]
An example of where this is required is the `accordion` component where the aria-controls on the button which expands the panel below requires a `aria-controls`. However, they cannot be linked because the constraints of the Shadow DOM
